lib/fs.ts

```diff
@@ -0,0 +1,631 @@
+'use strict'
+
+import { NoParamCallback, PathLike, WriteFileOptions, WriteStream } from 'fs'
+import { FSWatcher, WatchOptions } from 'chokidar'
+
+const BlueBirdPromise = require('bluebird')
```

Comment:
Mix ES6 import and CommonJS import will disable the type check for all `require`s.

Reply:
WIP